blog/source/_posts/MIDI-velocity-transformer.md

```diff
@@ -0,0 +1,266 @@
+---
+title: MIDI Velocity Prediction with Transformer
+date: 2023-09-11
+tags:
+author: wmatejuk
+---
+
+### Why is it important?
+
+{% algrtmImgBanner MIDI-velocity-transformer/banner-1.jpg pianoroll %}
+
+MIDI velocity is a crucial element in music dynamics, determining the force with which a note is played, 
+which profoundly influences the emotional quality of music. 
```

Comment:
I think you should try to explain the problem without referencing MIDI, and then you could introduce MIDI as a data structure related to this problem 🤔